obal/data/update_package.yml

```diff
@@ -5,12 +5,19 @@
   roles:
     - git_annex_setup
   tasks:
-    - name: 'Ensure upstream_files is defined'
+    - name: 'Fail if version not specified'
       fail:
-        msg: "'upstream_files' not defined for {{ inventory_hostname }}. Please define within package_manifest.yml"
-      when:
-        - upstream_files is not defined
-        - downstream_version is not defined
+        msg: "version must be specified"
+      when: version is not defined
+
+    - name: 'Git current branch'
+      command: 'git rev-parse --abbrev-ref HEAD'
+      register: git_branch
+
+    - name: 'Create bit branch'
+      command: "git checkout -b {{ git_branch.stdout }}-{{ inventory_hostname }}-{{ version }}"
```

Comment:
@ekohl I modified this to get the current branch and prepend that. So in theory someone might get:

```
master-tfm-ror51-rubygem-sinatra
rpm/develop-rubygem-foreman-tasks
master-pulp
```

Thoughts?

Reply:
@ehelms I meant a parameter the user could provide on the command line (or maybe read from the package manifest)?